tm2/pkg/sdk/auth/ante.go

```diff
@@ -153,7 +161,7 @@ func NewAnteHandler(ak AccountKeeper, bank BankKeeperI, sigGasConsumer Signature
 				// No signatures are needed for genesis.
 			} else {
 				// Check signature
-				signBytes, err := GetSignBytes(newCtx.ChainID(), tx, sacc, isGenesis)
+				signBytes, err := GetSignBytes(newCtx.ChainID(), tx, sacc, isGenesis || isNewAccount)
```

Comment:
we need to check the signatures; maybe with a different algorithm, but we cannot completely skip signature verification

Reply:
no, the logic here is able to verify the signatures unless the account is existed onchain before or not